core/modules/utils/filesystem.js

```diff
@@ -459,8 +479,11 @@ Delete a file described by the fileInfo if it exits
 */
 exports.deleteTiddlerFile = function(fileInfo, callback) {
 	//Only attempt to delete files that exist on disk
-	if(!fileInfo.filepath || !fs.existsSync(fileInfo.filepath)) {
-		return callback(null);
+	var invalid = !fileInfo.filepath || !fs.existsSync(fileInfo.filepath);
+	if(invalid) {
```

Comment:
did you add a variable here for easier debugging?

Reply:
Yes, I should put that back into the if, thank you.